src/views/illness/ill-person.njk

```diff
@@ -1,16 +1,95 @@
 {% extends 'layout.njk' %}
 
+{% from 'govuk/components/back-link/macro.njk'      import govukBackLink %}
+{% from 'govuk/components/error-summary/macro.njk'  import govukErrorSummary %}
+{% from 'govuk/components/input/macro.njk'          import govukInput %}
+{% from 'govuk/components/button/macro.njk'         import govukButton %}
+{% from 'govuk/components/radios/macro.njk'         import govukRadios %}
+
 {% block pageTitle %}
   Appeal due to ill health: Who was ill?
 {% endblock %}
 
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">
-        Who was ill?
-      </h1>
-      <p class="govuk-body">This page is currently in development.</p>
+
+      {{
+        govukErrorSummary ({
+          titleText: 'There is a problem with the details you gave us',
+          errorList: validationResult.errors
+        }) if validationResult and validationResult.errors.length > 0
+      }}
+
+      <form method="post">
+        {% set otherPersonHtml %}
+          {{ govukInput({
+            id: "other-person",
+            name: "otherPerson",
+            type: "text",
```

Comment:
within this components we usually use single quotes, could you fix this through the template. Html elements can remain with double quotes

Reply:
Good spot, fixed.